pages/create.js

```diff
@@ -1,10 +1,17 @@
 import NewOutivityForm from "@/components/NewOutivityForm";
 import { useRouter } from "next/router";
-
+import { useState } from "react";
 import { uid } from "uid";
 
 export default function CreateOutivity({ handleAddOutivity }) {
   const router = useRouter();
+  const [errorMessage, setErrorMessage] = useState("");
```

Comment:
It's good you are thinking about handling errors, but at the moment i don't see where you are using `setErrorMessage`. 

You could think about a `try catch` in your `createOutivity` function.